bridges/PanneauPocketBridge.php

```diff
@@ -0,0 +1,168 @@
+<?php
+class PanneauPocketBridge extends BridgeAbstract {
+	const NAME = 'Panneau Pocket';
+	const URI = 'https://app.panneaupocket.com';
+	const DESCRIPTION = 'Fetches the latest infos from Panneau Pocket';
+	const MAINTAINER = 'floviolleau';
+	const PARAMETERS = array(array(
+				'cities' => array(
+					'name' => 'Choisir une ville',
+					'type' => 'list',
+					'values' => self::CITIES
+				 )
+			   ));
+	const CACHE_TIMEOUT = 7200;
+
+	// debug function to get cities
+	private function getCities() {
+		$url = self::URI . '/public-api/city';
+		$ch = curl_init();
+		$curl_opts = [
+			CURLOPT_URL => $url,
+			CURLOPT_USERAGENT => 'Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:80.0) Gecko/20100101 Firefox/80.0',
+			CURLOPT_RETURNTRANSFER => 1,
+			CURLOPT_SSL_VERIFYPEER => 0,
+			CURLOPT_FOLLOWLOCATION => 1,
+		];
+		$content = getContents($url, [], $curl_opts);
+		$cities = json_decode($content, true);
+
+		$formattedCities = null;
+		$citiesString = '[<br>';
+		foreach($cities as $city) {
+			if (str_starts_with($city['postCode'], '35')) {
+				$formattedCities[$city['name'] . ' - ' . $city['postCode']] = $city['id'];
+				$citiesString .= '"' . $city['name'] . '-' . $city['postCode'] . '" => "' . $city['id'] . '",';
+				$citiesString .= '<br>';
+			}
+		}
+		$citiesString .= ']';
+		echo '<pre>' . $citiesString . '</pre>';
+		die();
+	}
+
+	private function cleanTitle($string) {
+		$string = str_replace(' ', '-', $string);
+		return preg_replace('/[^A-Za-z0-9\-]/', '', $string);
+	}
+
+	public function collectData() {
+		// for debug
+		// $this->getCities();
+		$matchedCity = array_search($this->getInput('cities'), self::CITIES);
+		$city = strtolower($this->getInput('cities') . '-' . $matchedCity);
+		$url = self::URI . '/ville/' . $city;
+
+		$html = getSimpleHTMLDOM($url);
+		$itemsDom = $html->find('.sign-carousel--item');
+		foreach($itemsDom as $itemDom) {
+			$publicationUrl = $itemDom->find('a.dropdown-item.item-twitter', 0)->href;
+			$publicationUrl = str_replace('https://twitter.com/intent/tweet?url=', '', $publicationUrl);
+			$publicationUrl = str_replace('&text=', '', $publicationUrl);
+			$publicationTitle = $itemDom->find('.sign-preview__content .title', 0)->innerText();
```

Comment:
Same for the other innerText()
```suggestion
			$publicationTitle = $itemDom->find('.sign-preview__content .title', 0)->innertext;
```

Reply:
innerText do not work while innerText() works